tests/WikibaseRdfIntegrationTest.php

```diff
@@ -17,6 +17,11 @@
 
 class WikibaseRdfIntegrationTest extends MediaWikiIntegrationTestCase {
 
+	protected function setUp(): void {
+		parent::setUp();
+		$this->tablesUsed[] = 'page';
```

Comment:
This is what fixes the cleanup.

When I originally looked at the following code:

![image](https://user-images.githubusercontent.com/146040/181934068-379086b5-2d30-4632-a1dc-e10de5e7ae3e.png)

I thought that these tables were getting added automatically. But no, you still need to provide the "main" one.